backend/coins/btc/electrum/electrum.go

```diff
@@ -139,9 +123,15 @@ func NewElectrumConnection(servers []*rpc.ServerInfo, log *logrus.Entry, dialer
 	log = log.WithFields(logrus.Fields{"group": "electrum", "server-type": "electrumx", "servers": serverList})
 	log.Debug("Connecting to Electrum server")
 
-	backends := []rpc.Backend{}
+	backends := []*jsonrpc.Backend{}
```

Comment:
`var []*jsonrpc.Backend` would make it slightly easier to read.
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Done